packages/atlas/src/providers/joystream/joystream.hooks.ts

```diff
@@ -155,15 +161,31 @@ export const useSubscribeAccountBalance = (controllerAccount?: string | null) =>
     const init = async () => {
       unsubscribe = await joystream.subscribeAccountBalance(
         controllerAccount || activeMembership.controllerAccount,
-        proxyCallback((balance) => setAccountBalance(new BN(balance)))
+        proxyCallback((balance) => {
+          if (opts?.isRewardAccount) {
+            // substract existential deposit from channel balance
+            // TODO in future existentialDeposit will be replaced with channel state bloat bond
+            const rewardBalance = new BN(balance).sub(chainState.existentialDeposit)
+            setAccountBalance(rewardBalance)
+            return
+          }
```

Comment:
I think first you need to confirm that the account actually has any funds. Because (at least currently, without the channel state bloat bond being in the account) new channels will have initial balance of 0 (the account doesn't really "exist" until it gets it first income). So for new channels the balance we display will actually be negative (0 - existential deposit):
<img width="274" alt="CleanShot 2022-08-25 at 16 26 36@2x" src="https://user-images.githubusercontent.com/12646744/186692455-888a04aa-6699-4f2d-a98f-014db483e5f3.png">

(-0 because of rounding most likely)